app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationsListItem/integrationsListItem.scss

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.integrations-list-item {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  width: 820px;
+  min-height: 104px;
```

Comment:
Usually we are not set strict height it should depends on the content ask UX team we are have or have no min height